### PR TITLE
feat(ddm): summary table

### LINKED ISSUE #55653: Add summary table underneath graph

### PR DESCRIPTION
**Pull request by obostjancic**
Adds a summary table underneath the chart. Closes #55653
image

### CODE REVIEW

File: static/app/views/ddm/metricsExplorer.tsx [hunk 1]
```diff
@@ -399,131 +435,73 @@ function normalizeChartTimeParams(data: MetricsData) {
...
-          <AreaChart {...chartProps} {...zoomRenderProps} />
-        ) : (
-          <BarChart stacked {...chartProps} {...zoomRenderProps} />
-        )
-      }
-    </ChartZoom>
-  );
-}
-
-function Table({data}: {data: MetricsData}) {
```
Comment (obostjancic) [inline static/app/views/ddm/metricsExplorer.tsx:480]: This was not used much, we can bring it back if really needed

File: static/app/views/ddm/metricsExplorer.tsx [hunk 2]
```diff
@@ -326,15 +334,43 @@ function MetricsExplorerDisplay({displayType, ...metricsDataProps}: DisplayProps
     );
   }
 
+  // TODO(ddm): we should move this into the useMetricsData hook
   const sorted = sortData(data);
+  const unit = getUnitFromMRI(Object.keys(data.groups[0].series)[0]); // this assumes that all series have the same unit
```
Comment (matejminar) [inline static/app/views/ddm/metricsExplorer.tsx:339]:
```suggestion
  const unit = getUnitFromMRI(Object.keys(data.groups[0]?.series ?? {})[0]); // this assumes that all series have the same unit
```
I changed this recently ^

File: static/app/utils/metrics.tsx [hunk 3]
```diff
@@ -263,3 +263,15 @@ export function formatMetricUsingUnit(value: number | null, unit: string) {
       return value.toLocaleString();
   }
 }
+
+export function formatMetricsUsingUnitAndOp(
```
Comment (matejminar) [inline static/app/utils/metrics.tsx:267]: ❤️

File: static/app/views/ddm/summaryTable.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  operation?: string;
+}) {
+  return (
+    <SummaryTableWrapper>
+      <HeaderCell />
+      <HeaderCell>Name</HeaderCell>
+      <HeaderCell>Avg</HeaderCell>
+      <HeaderCell>Min</HeaderCell>
+      <HeaderCell>Max</HeaderCell>
+      <HeaderCell>Sum</HeaderCell>
```
Comment (matejminar) [inline static/app/views/ddm/summaryTable.tsx:24]: Let's translate these and start using the translate function from now on on everything. I fixed all occurrences that were without so we should have a clean slate.

**Review by matejminar — APPROVED** [review]

**Pull request merged**